pkgs/tools/misc/coreutils/default.nix

```diff
@@ -33,6 +33,8 @@ stdenv.mkDerivation (rec {
     ./fix-chmod-exit-code.patch
     # Workaround for https://debbugs.gnu.org/cgi/bugreport.cgi?bug=51433
     ./disable-seek-hole.patch
+    # Workaround for https://debbugs.gnu.org/cgi/bugreport.cgi?bug=52330
```

Comment:
I would just add a comment here to drop once version xxx gets released (just not to forget).

Reply:
Done, though I didn't specify the version number as that information is not decided yet.